java/fory-core/src/main/java/org/apache/fory/config/Config.java

```diff
@@ -297,6 +297,14 @@ public boolean isScalaOptimizationEnabled() {
     return scalaOptimizationEnabled;
   }
 
+  /**
+   * Whether use meta share for ObjectStreamSerializer. This reduces serialization size by using
+   * ObjectSerializer instead of CompatibleSerializer. Only effective when metaShareEnabled is true.
+   */
+  public boolean isMetaShareForObjectStreamEnabled() {
```

Comment:
We don't need this, we can just check whether fory compatible mode is set to `CompatibleMode` in `ObjectStreamSerializer`